### DISCUSSION TITLE
How to resolve webrtc H264

### DISCUSSION THREAD

**Discussion by KodaGao** [cefsharp/CefSharp#3721] [General]
I normally use webrtc in edge and chrome browsers, but the RSTP service reports an error in cefsharp.winform v91.1.230, prompting an error in the write header。

use cefsharp.winform v91.1.230， Prompt in RSTP service the errors message is “ remote does not support encoding”，I think is the H264 is not support 。How to resolve？

**Comment by amaitland** [comment]
English only please. No images of errors, please provide the error in text format. 

https://idownvotedbecau.se/imageofanexception/

**Reply by KodaGao** [reply]
use cefsharp.winform v91.1.230， Prompt in RSTP service the errors message is “ remote does not support encoding”，I think is the H264 is not support 。How to resolve？

**Reply by amaitland** [reply]
See https://github.com/cefsharp/CefSharp/wiki/General-Usage#multimedia-audiovideo
